flytekit/clis/sdk_in_container/run.py

```diff
@@ -256,8 +255,14 @@ class RunLevelParams(PyFlyteParams):
     _remote: typing.Optional[FlyteRemote] = None
 
     def remote_instance(self) -> FlyteRemote:
+        # TODO we have to check if the previous remote was for the same data upload location
         if self._remote is None:
-            self._remote = get_remote(self.config_file, self.project, self.domain)
+            # TODO @wild-endeavor - why should the local data upload location be /tmp? what should it be?
+            # Also why do we even copy the local data?
+            data_upload_location = "/tmp"
```

Comment:
Do we need to use random paths? [As in this case](https://github.com/flyteorg/flytekit/blob/1c813759b2c56d4821350918ee0aef9a78d86859/flytekit/core/data_persistence.py#L498-L501)

Reply:
done